backend/controllers/cellMove/cellNotAvailable.ts

```diff
@@ -1,6 +1,6 @@
 import { putLastNameFirst } from '../../utils'
 
-export default ({ prisonApi }) =>
+export default ({ prisonApi, logError }) =>
```

Comment:
Same as per other PR (https://github.com/ministryofjustice/digital-prison-services/pull/2219) - use underscore?

Reply:
i have been told it causes warnings in this case, so we'll leave it as is